core/includes/updater.inc

```diff
@@ -144,6 +144,11 @@ abstract class Updater implements BackdropUpdaterInterface {
       return FALSE;
     }
     foreach ($info_files as $info_file) {
+      // Exclude info files for tests.
+      if (backdrop_substr($info_file->filename, -10) == 'tests.info') {
```

Comment:
This would prevent a contrib module named "footests" to be found, isn't it ?

Reply:
Good point @opi. I modified it slightly to include leading period and lengthened it to 11 characters.